src/bin/zos-cli.js

```diff
@@ -1,8 +1,18 @@
 #! /usr/bin/env node
 
 import { Logger } from 'zos-lib'
+import { lockSync } from 'lockfile';
 import program from './program'
 
+// Acquire file lock to ensure no other instance is running
+const LOCKFILE = '.zos.lock';
+try { 
+  lockSync(LOCKFILE, { retries: 0 }) 
+} catch (e) { 
+  console.error(`Could lock file '${LOCKFILE}'. Ensure no other instance of zos is running.`); 
+  process.exit(1); 
+}
```

Comment:
Don't we need to clean the file?

Reply:
Nope, `lockfile` takes care of everything :-)